scripts/setup_solr.sh

```diff
@@ -38,7 +38,19 @@ cp $REPODIR/../config/solr/managed-schema .
 echo $SUDOPASS | sudo -S -k chmod 775 managed-schema
 echo "Schema fetched."
 
-echo "Creating a ckan core..."
-echo $SUDOPASS | sudo -S -k -u solr /opt/solr/bin/solr create -c ckan
-echo "ckan core created."
+# New CKAN core creation method
+echo "Updating CKAN core with new method..."
+sudo -u solr /opt/solr/bin/solr stop -all
```

Comment:
should be `echo $SUDOPASS | sudo -S -k` format rather than `sudo`

Reply:
changed